### DISCUSSION TITLE
multiple values for up-target: working in 1.0.0, not in 2.2.1?

### DISCUSSION THREAD

**Discussion by matthewn** [unpoly/unpoly#250] [General]
Greetings, and first let me express my gratitude for Unpoly, which is just splendid.

I am trying to upgrade a site from 1.0.0 to 2.2.1, and have encountered the following trouble. I have numerous links that have multiple values for the up-target attribute, like this:

`<a href="/blog/" up-target="#header, #navbar, #content, .wagtail-userbar-items" up-transition="cross-fade">blog</a>`

These links work as expected in 1.0.0 -- that is, they update all the elements listed in `up-target`.

In 2.2.1, only the page's `<main>` -- or whichever element I put `up-main` on -- gets updated. No errors in my console, and up.log.enable() sheds no light on the situation -- it just confirms that `up-render` is 'Updating "main" in root layer'.

Is there something I am missing to prevent 2.2.1 from targetting `<main>` or `up-main` and instead honor the values in `up-target`?

**Comment by adam12** [comment]
Are you able to recreate it using the example [Glitch](https://glitch.com/edit/#!/empty-unpoly-project) app? And could you share the full log from `up.log.enable()`?

Are you using layers?

**Reply by matthewn** [reply]
Thank you kindly for the reply. As I dug back into this to answer your questions, I discovered the answer, and as I suspected, it's not Unpoly's fault at all -- this is a Django/Wagtail project I'm using Unpoly with, and I had forgotten there was css/js compression happening before the code goes out on the wire. When I turn that off, Unpoly behaves as expected. :)

**Comment by triskweline** [comment]
The upcoming version 2.3 prints a log message when the target cannot be found and a fallback target is used instead.
